cmd/kubelet/app/server.go

```diff
@@ -651,6 +653,7 @@ type KubeletConfig struct {
 	Auth                           server.AuthInterface
 	Builder                        KubeletBuilder
 	CAdvisorInterface              cadvisor.Interface
+	CalculateVolumeDiskUsageFrequency time.Duration
```

Comment:
I know the name sounds a bit weird, but I think it makes sense as a duration is just a length of time.

From the docs:
>
A Duration represents the elapsed time between two instants

This seems like what we want.